easybuild/easyconfigs/l/LncLOOM/LncLOOM-2.0-foss-2020b.eb

```diff
@@ -0,0 +1,71 @@
+easyblock = 'PythonBundle'
+
+name = 'LncLOOM'
+version = '2.0'
+
+homepage = 'https://github.com/lncLOOM/lncLOOMv2/'
+description = """LncLOOM is a graph-based framework that uses integer
+programming to identify combinations of short motifs that are deeply conserved
+in rapidly evolving sequences."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('SciPy-bundle', '2020.11'),
+    ('MAFFT', '7.475', '-with-extensions'),
+    ('BLAST+', '2.11.0'),
+    ('Gurobi', '9.1.1'),
+    ('Biopython', '1.78'),
+]
+
+use_pip = True
+
+exts_default_options = {'source_urls': [PYPI_SOURCE]}
+exts_list = [
+    ('amply', '0.1.4', {
+        'checksums': ['cb12dcb49d16b168c02be128a1527ecde50211e4bd94af76ff4e67707f5a2d38'],
+    }),
+    ('networkx', '2.5', {
```

Comment:
Please include this as a proper dependency, we already have a suitable `networkx` easyconfig for `foss/2020b`